`gosec G122` lint failure in `doc/man-docs/man_doc.go`

The latest golangci-lint image introduces `gosec v2.24`, which adds a new rule `G122` , detecting `TOCTOU (Time-of-Check/Time-of-Use)` race conditions.

This causes the lint pipeline to fail on main:

```
doc/man-docs/man_doc.go:71:31: G122: Filesystem operation in filepath.Walk/WalkDir callback uses race-prone path; consider root-scoped APIs (e.g. os.Root) to prevent symlink TOCTOU traversal (gosec)            
                        content, err := os.ReadFile(path)                                                
                                                   ^                                                     
doc/man-docs/man_doc.go:93:22: G122: Filesystem operation in filepath.Walk/WalkDir callback uses race-prone path; consider root-scoped APIs (e.g. os.Root) to prevent symlink TOCTOU traversal (gosec)            
                        err = os.WriteFile(path, []byte(cleanedContent), 0600)   

```

--- a/doc/man-docs/man_doc.go
+++ b/doc/man-docs/man_doc.go
@@ -106,7 +106,7 @@ func cleanManPages(docDir string) error {
 			cleanedContent := strings.Split(updatedContent, "\n.SH HISTORY")[0]
 			cleanedContent = strings.TrimRight(cleanedContent, "\n")
 
-			wf, err := root.OpenFile(relPath, os.O_WRONLY|os.O_TRUNC, 0600)
+			wf, err := root.OpenFile(relPath, os.O_WRONLY|os.O_TRUNC|os.O_CREATE, 0600)
 			if err != nil {
 				return err
 			}

